### PR TITLE
Support for PostgreSQL 11

### LINKED ISSUE #1244: Support PostgreSQL 11
### What problem are you facing?
Missing PostgreSQL 11 support

### How could Crossplane help solve your problem?
Allow PostgreSQL 11 support

### PR DESCRIPTION
**Pull request by vasartori**
### Description of your changes

Allow the version `11` on field `engineVersion` in resource `PostgreSQLInstance`
Fixes #1244

### How has this code been tested?

Create a PostgreSQLInstance spec with this:
```yaml
apiVersion: database.crossplane.io/v1alpha1
kind: PostgreSQLInstance
metadata:
  name: database-test
spec:
  classSelector:
    matchLabels:
      example: "true"
  writeConnectionSecretToRef:
    name: database-test
  engineVersion: "11"
```
### Checklist

I have:
- [x] Run `make reviewable` to ensure this PR is ready for review.
- [x] Ensured this PR contains a neat, self documenting set of commits.
- [x] Updated any relevant [documentation] and [examples].
- [ ] Reported all new error conditions into the log or as an event, as
  appropriate.

For more about what we believe makes a pull request complete, see our
[definition of done].

[documentation]: https://github.com/crossplaneio/crossplane/tree/master/docs
[examples]: https://github.com/crossplaneio/crossplane/tree/master/cluster/examples
[definition of done]: https://github.com/crossplaneio/crossplane/tree/master/design/one-pager-definition-of-done.md

### CODE REVIEW

File: docs/api/crossplaneio/crossplane/database-crossplane-io-v1alpha1.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ Appears in:
 
 Name | Type | Description
 -----|------|------------
-`engineVersion` | string | EngineVersion specifies the desired PostgreSQL engine version, e.g. 9.6.
+`engineVersion` | string | EngineVersion specifies the desired PostgreSQL engine version. 9.6 or 11 are allowed.
```
Comment (hasheddan) [inline docs/api/crossplaneio/crossplane/database-crossplane-io-v1alpha1.md]: These docs are auto-generated using https://github.com/negz/crossdocs, so this should reflect the code comment on the `engineVersion` field. Unfortunately, it is a bit difficult for to setup `crossdocs` currently, so I see the following options here:
- You could remove this change to the docs and opt for the existing comment (it isn't wrong currently, but your update is an improvement IMO).
- You could setup `crossdocs` (I am more than happy to help here), and you could modify the code comment, then regenerate the docs here.
- You could remove this change, then I would be happy to follow-up with a PR to change the code comment and regenerate the docs. I could add you as an approver to make sure it matches your expectations.

This is a poor part of the development workflow right now and we need to move on getting a fix for it (related issue:  https://github.com/crossplaneio/crossplane/issues/1166). Apologies for the complications it is currently causing!

File: apis/database/v1alpha1/types.go [hunk 2]
```diff
@@ -66,7 +66,7 @@ type PostgreSQLInstanceSpec struct {
 	runtimev1alpha1.ResourceClaimSpec `json:",inline"`
 
 	// EngineVersion specifies the desired PostgreSQL engine version, e.g. 9.6.
```
Comment (hasheddan) [inline apis/database/v1alpha1/types.go]: This is the code comment mentioned below from which the docs are generated.

Comment (vasartori) [reply apis/database/v1alpha1/types.go]: @hasheddan I think the best approach is setup crossdocs.
So, I tryed to run crossdocs without success... I folow the readme, but a lot of errors appears... 

I think my mistake with crossdocs was in this step: ```Update Gopkg.toml and doc.sh to ensure you're vendoring and running documentation generation for the versions of the Crossplane projects you care about.```

Comment (hasheddan) [reply apis/database/v1alpha1/types.go]: I'll follow-up with you offline to get you set up :+1:

**Comment by vasartori** [comment]
Done!

**Comment by negz** [comment]
Thanks @vasartori!

**Pull request merged**